src/Surfnet/ServiceProviderDashboard/Application/ViewObject/EntityDetail.php

```diff
@@ -253,6 +254,11 @@ class EntityDetail
      */
     private $isPublicClient;
 
+    /**
+     * @var string[]|null
+     */
+    private $resourceServers = null;
```

Comment:
Why not use the `ResourceServerCollection` and wouldn't it be better to use a concrete implementation then to use a variable that could be a string array or null.

Reply:
The existing ResourceServerCollection is used in the context of selecting resource servers on the form. In this case we are dealing with fully equipped Manage entities that are used for display purposes.

I agree, it would be best to also pour these into more dedicated view object. But not reusing the existing objects.

Reply:
I think it woul d be better to don't return null but always a collection or array. This will prevent the necessity of validation when this method is being used. Feel free to leave it as-is for now.